doc/lvm.md

```diff
@@ -0,0 +1,168 @@
+
+Note that, unlike RAID0, stripped LVs are not really a separate type. Many types of LVs can be
+stripped.
+
+### Normal LVM
+
+This is somehow the base case. Normal LVs are visualized in the standard way and all operations
+are supported.
+
+### Thin Pool and Thin LV
+
+#### How is it displayed?
+
+- Only the thin pool and its thin LVs are displayed. No trace of the hidden LVs used to store the
+  data and the metadata of each thin LV or to store the spare metadata of the VG.
+- Due to a bug, **nothing in the UI identifies the displayed LVs as being special**. They basically
+  look like normal LVs, although `BlkDevicesTable::DEVICE_LABELS` contains entries for both thin
+  pools and thin LVs.
+- In LVM is not possible to define stripping for thin LVs, they use the stripping defined for their thin
+  pools. The partitioner UI **reports 0 stripes for all thin LVs**.
+
+#### What can be done?
+
+- For thin pools
+  - Create: it works.
+  - Edit (format/mount): not allowed ("_Is a thin pool. It cannot be edited_").
+  - Resize: it works. Thin pools already in disk cannot be shrinked, which is correct.
+  - Delete: it works. Note it deletes the pool, all its thin volumes and the associated hidden LVs.
+
+- For thin LVs
+  - Create: it works. The **widgets for defining stripping are disabled and set to the default values**.
+    Maybe it would be better to show them disabled but with the pool values. Or to not show them at all.
+  - Edit (format/mount): not allowed ("_Is a thin pool. It cannot be edited_").
+  - Resize: it works.
+  - Delete: it works. Not it deletes the thin volume and all the associated hidden LVs.
+
+### Cache LV with a Cache Pool
+
+#### How is it displayed?
+
+- Only the cache LV is displayed. No trace of the hidden LVs (origin LV, cache pool, cache data LV
+  nor cache metadata LV).
+- **Nothing in the UI identifies the cache LV as being so**, it basically looks like a normal LV
+  (eg. there is no entry for cache LV at `BlkDevicesTable::DEVICE_LABELS`)
+
+#### What can be done?
+
+- Create: not possible.
+- Edit (format/mount): just as a normal LV.
+- Resize: not allowed ("_Resizing of this type of LVM logical volumes is not supported_").
+- Delete: it works. Note it deletes everything: the cache LV, its origin LV, the cache pool, its cache
+  data LV and its cache metadata LV.
+
+### Cache LV with a Cache Volume
+
+#### How is it displayed?
+
+- Only the cache LV is displayed. No trace of the hidden cache volume.
+- **Nothing in the UI identifies the cache LV as being so**, just like the previous case.
+
+#### What can be done?
+
+- Create: not possible.
+- Edit (format/mount): just as a normal LV.
+- Resize: not allowed ("_Resizing of this type of LVM logical volumes is not supported_").
+- Delete: it works. It only deletes the cache LV, the LV used as cache survives (it becomes a
+  normal LV) **but is not visible immediately**. A reprobing after the commit phase is needed.
```

Comment:
I consider the behaviour of LVM a bug and reported it, see https://bugzilla.suse.com/show_bug.cgi?id=1171907.

Reply:
Link to bugzilla added.